addon/components/paper-switch.js

```diff
@@ -1,3 +1,5 @@
+/* globals Hammer */
```

Comment:
I added it to the global `.jshintrc` file in c80fb32, so we don't have to add it to individual files in the future.